package.json

```diff
@@ -60,7 +60,7 @@
     "cozy-flags": "2.1.0",
     "cozy-realtime": "3.7.0",
     "cozy-scripts": "5.1.1",
-    "cozy-sharing": "2.10.0",
+    "cozy-sharing": "^3.3.1",
```

Comment:
Is it expected that the dependency is not pinned ? Just to be sure, the breaking change that triggered 2 to 3 is not problematic in notes ?

Reply:
Nop, already migrated to mui-v4 so not affected by the BC and thanks for the pin!